tests/test_notebooks.py

```diff
@@ -6,13 +6,19 @@
 from nbconvert.preprocessors import ExecutePreprocessor
 
 import sys, os
+import subprocess
 myPath = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, myPath + '/../')
 
 
 def notebook_test_function(name):
 
-    with open(name) as f:
+    env_name = str(subprocess.run("which python", shell=True,capture_output=True).stdout)
```

Comment:
`which python` will only work on linux/osx. Leave the environment hardcoded, we will document it so in the "Local development environment setup and testing" guide.